internal/terraform/rootmodule/walker.go

```diff
@@ -103,8 +103,14 @@ func (w *Walker) IsWalking() bool {
 	return w.walking
 }
 
-func (w *Walker) walk(ctx context.Context, rootPath string, wf WalkFunc) error {
+func (w *Walker) walk(ctx context.Context, rootPath string, excludePath []string, wf WalkFunc) error {
 	defer w.Stop()
+
+	excludePathMap := make(map[string]bool)
+	for _, path := range excludePath {
+		excludePathMap[ToLowerVolumePath(path)] = true
```

Comment:
Thanks for adding the sanitazition but wouldn't it be better to do this in the initialize handler (`resolvePath`), where we also resolve symlinks, homedir etc.? I feel that the walker should just expect already sanitized input.